applications/content-creator-applications/controller/contentCreatorApplicationController.js

```diff
@@ -53,17 +53,16 @@ module.exports = function makeContentCreatorApplicationController({ contentCreat
 
 			const validCCApplication = makeContentCreatorApplication(applicationInfo);
 
-			const created = await contentCreatorApplicationList.add({
-				id: validCCApplication.getId(),
-				firstName: validCCApplication.getFirstName(),
-				lastName: validCCApplication.getLastName(),
-				email: validCCApplication.getEmail(),
-				motivation: validCCApplication.getMotivation(),
-				approved: validCCApplication.isApproved(),
-				rejectionReason: validCCApplication.getRejectReason(),
-				createdAt: validCCApplication.getCreatedAt(),
-				modifiedAt: validCCApplication.getModifiedAt(),
-			});
+            const created = await contentCreatorApplicationList.add({
+                id: validCCApplication.getId(),
+                name: validCCApplication.getName(),
```

Comment:
Isn't name supposed to be divided into first name and last name?

Reply:
it only uses name for now. we will update this file to contain first name and last name in a future sprint, but for now we only use name